rust/main/Cargo.toml

```
@@ -148,6 +149,7 @@ tokio-metrics = { version = "0.3.1", default-features = false }
 tokio-test = "0.4"
 toml_edit = "0.19.14"
 tonic = "0.12.3"
+tower = "0.5.2"
```

I believe `tower` is already in the dependency tree, can probably set to 
```suggestion
tower = "*"
```